### ISSUE TITLE
домен sportspace

### ISSUE THREAD

**Issue by zuhijan** [sportspace-develop/front-next#44]

**Comment by playmixer** [comment]
Возможно эту задачу нужно расширить до создания хоста, выпуск сертификата и настройки этого всего?

**Comment by zuhijan** [comment]
> Возможно эту задачу нужно расширить до создания хоста, выпуск сертификата и настройки этого всего?

Да, согласен полностью. Просто налету создал, пока разговаривали. Но пока это даже наверное не в МВП. Мы вроде хотели отложить хост и прочее?
